src/gui/application.cpp

```diff
@@ -123,10 +123,9 @@ Application::Application(Platform *platform, const QString &displayLanguage, boo
 
 #ifdef WITH_AUTO_UPDATER
     // Update checks
-    UpdaterScheduler *updaterScheduler = new UpdaterScheduler(this);
```

Comment:
why the change in ownership?

Reply:
I just rebased the PR...

Reply:
Because we need access to settings dialog to correctly parent the update finished dialog. It shouldn't matter who the owner is.

Reply:
Its a QObject so the parenting decides on the lifetime, I'd suggest to add an additional parameter.